Source/Urho3D/Engine/Console.cpp

```diff
@@ -376,14 +376,10 @@ void Console::HandleTextFinished(StringHash eventType, VariantMap& eventData)
         // Send the command as an event for script subsystem
         using namespace ConsoleCommand;
 
-#if URHO3D_CXX11
-        SendEvent(E_CONSOLECOMMAND, P_COMMAND, line, P_ID, static_cast<Text*>(interpreters_->GetSelectedItem())->GetText());
-#else
         VariantMap& newEventData = GetEventDataMap();

```

Comment:
Here it is
https://discourse.urho3d.io/t/aggregate-initialization-with-urho3d-containers/1901
I re-thinked the subject... I am still against merging all event parameters into the single line. This is like with variable declarations, you know.
However, I think now that variadic `SendEvent` is better than non-variadic because it doesn't expose internal map to sender.